### ISSUE TITLE
Targets specified in prebuildrc are squashed into one

### ISSUE THREAD

**Issue by oleavr** [mafintosh/prebuild#80]
I'm not sure if this is a bug in prebuild, but this no longer seems to work as it used to:
https://github.com/frida/frida-node/commit/b1183e51aa9851d8fd9c5d792740edd36d4bc914

This results in:
```
prebuild ERR! configure error
prebuild ERR! stack Error: Invalid version number: 4.0.0,5.0.0
prebuild ERR! stack     at getNodeDir (/Users/oleavr/src/frida/frida-node/node_modules/node-gyp/lib/configure.js:69:25)
prebuild ERR! stack     at /Users/oleavr/src/frida/frida-node/node_modules/node-gyp/lib/configure.js:41:7
prebuild ERR! stack     at /Users/oleavr/src/frida/frida-node/node_modules/node-gyp/lib/configure.js:393:9
prebuild ERR! stack     at ChildProcess.exithandler (child_process.js:204:7)
prebuild ERR! stack     at emitTwo (events.js:87:13)
prebuild ERR! stack     at ChildProcess.emit (events.js:172:7)
prebuild ERR! stack     at maybeClose (internal/child_process.js:818:16)
prebuild ERR! stack     at Socket.<anonymous> (internal/child_process.js:319:11)
prebuild ERR! stack     at emitOne (events.js:77:13)
prebuild ERR! stack     at Socket.emit (events.js:169:7)
prebuild ERR! not ok
```

I dumped the arguments of that function and it gave me:
```
semver.valid(argv[0]): null
argv[0]: true
gyp.opts.target: 4.0.0,5.0.0
defaultVersion: v5.3.0
```

So it seems like prebuild is asking node-gyp to build for `4.0.0,5.0.0`.

**Comment by juliangruber** [comment]
ew, I hope that's not related to my recent refactorings. I'll take a look

**Comment by juliangruber** [comment]
found the problem, fixing

**Comment by juliangruber** [comment]
so the issue is that `prebuild --install --target=X` was required for electron, however with a prebuildrc in place there's multiple targets and it doesn't, as before, just build for the current node version.

**Comment by juliangruber** [comment]
the values from .prebuildrc should only be used for prebuilding, not for installing. i think the way to go is to split `--target` into two options, one for the install command and one for prebuilding. what do others think?

**Comment by mafintosh** [comment]
@juliangruber sounds good

**Comment by ralphtheninja** [comment]
> the values from .prebuildrc should only be used for prebuilding, not for installing. i think the way to go is to split --target into two options, one for the install command and one for prebuilding. what do others think?

+1

**Comment by ralphtheninja** [comment]
@juliangruber Can we add a simple test for this?

**Comment by juliangruber** [comment]
I'll be on vacation without internet starting tomorrow, you guys will have to do this without me. Sorry for leaving it in this state!

**Comment by ralphtheninja** [comment]
@oleavr This should be fixed now in 3.0.0
